dragonchain/broadcast_processor/broadcast_functions.py

```diff
@@ -176,6 +196,27 @@ def schedule_block_for_broadcast_sync(block_id: str, time: int = 0) -> None:
     await redis.zadd_async(IN_FLIGHT_KEY, time, block_id)
 
 
+def schedule_notification_for_broadcast_sync(notification_location: str, time: int = 0) -> None:
```

Comment:
should remove time from this function as it's now irrelevant.

Reply:
done